crates/db_schema/src/main.rs

```diff
@@ -0,0 +1,9 @@
+use lemmy_utils::error::LemmyError;
+
+/// Runs schema setup
+#[tokio::main]
+async fn main() -> Result<(), LemmyError> {
+  lemmy_db_schema::utils::build_db_pool().await?;
+
+  Ok(())
+}
```

Comment:
This is only for CI? Seems useless as it needs to do a full build anyway.

Reply:
Should be easy to apt install no?

Reply:
I will try to make lemmy_server use rustls only, instead of both rustls and dynamically linked libssl

Reply:
Setting `pull: true` in .woodpecker.yml fixed it. The changes to rustls will be reverted so the decision can be made separately from this PR.

Reply:
Fixed it by not using diesel-cli image